frontend/__tests__/unit/pages/About.test.tsx

```diff
@@ -35,6 +35,78 @@ jest.mock('utils/aboutData', () => ({
     'This is a test paragraph about the project.',
     'This is another paragraph about the project history.',
   ],
+  leaders: {
+    arkid15r: 'CCSP, CISSP, CSSLP',
+    kasya: 'CC',
+    mamicidal: 'CISSP',
+  },
+  missionContent: {
+    mission:
+      'OWASP Nest is a comprehensive platform designed to empower developers, security professionals, and organizations in their journey toward building secure applications.',
+    whoItsFor:
+      'OWASP Nest is designed for developers, security professionals, project leaders, and organizations looking to integrate security best practices into their development workflows.',
+  },
+  keyFeatures: [
+    {
+      title: 'Advanced Search Capabilities',
+      description:
+        'Find OWASP projects, tools, and resources with precision using our intelligent search functionality.',
+    },
+    {
+      title: 'Slack Integration',
+      description: 'Stay connected with the OWASP community through seamless Slack integration.',
+    },
+    {
+      title: 'OWASP Chapters Proximity Page',
+      description:
+        'Discover local OWASP chapters and events near you to engage with the community.',
+    },
+    {
+      title: 'AI-Generated Insights',
+      description:
+        'Leverage AI-powered insights to better understand project relationships and recommendations.',
+    },
+  ],
+  getInvolvedContent: {
+    description:
+      'OWASP Nest thrives thanks to community-driven contributions. Here are ways you can get involved:',
+    ways: [
+      'Code Contributions – Fix bugs or build new features.',
+      'Code Review – Improve quality by reviewing pull requests.',
+      'Documentation – Help improve our guides and tutorials.',
+      'Testing – Report bugs and help test new features.',
+      'Community Engagement – Share your experiences and help others.',
+    ],
+    callToAction:
+      'To get started, visit the OWASP Nest Repository, explore the Contributing Guidelines, and review the Code of Conduct.',
+  },
+  projectHistory: [
```

Comment:
This one looks inconsistent w/ the rest of the content too. I don't get your point of having multiple styles on the same page 🤷‍♂️ 

![Screenshot 2025-06-16 at 8 43 06 PM](https://github.com/user-attachments/assets/b1967f1b-9ac6-4d4b-bb94-7202ecc7b7bd)

Reply:
Same for the white theme

![Screenshot 2025-06-16 at 8 45 13 PM](https://github.com/user-attachments/assets/b7ef7d05-1342-4ae3-a2d5-91391c2c0c44)